apps/video/index.html

```diff
@@ -51,68 +51,70 @@
     <script defer src="js/video.js"></script>
   </head>
   <body role="application" class="skin-dark theme-media">
-    <div id="two-column-spearator"></div>
-    <section role="region" id="thumbnail-list-view">
-      <gaia-header id="picker-header" action="back">
-        <h1 id="picker-title" data-l10n-id="pick-title"></h1>
-        <h1 id="thumbnail-list-title">Video</h1>
-      </gaia-header>
+    <div id="container">
```

Comment:
Nit: The readability of the html would be improved by using a more descriptive name for "container". Some possibilities: "thumbnail-player-container", "non-overlay-container", "app-container".